lib/Server.js

```diff
@@ -641,11 +643,10 @@ class Server {
   }
 
   setContentHeaders(req, res, next) {
-    if (this.headers) {
-      // eslint-disable-next-line
-      for (const name in this.headers) {
-        res.setHeader(name, this.headers[name]);
-      }
+    if (this.options.headers) {
```

Comment:
Let's wait this too because developers can change `this.headers` in runtime and it is break logic